src/stringBuffer.js

```diff
@@ -24,7 +24,10 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let result = '';
+  return function(value) {
+    return value !== undefined ? (result += value) : result;
```

Comment:
`(result += value)` - it is tricky. If you need to do 2 operations(update result and return updated value) inside ternary will be better to use `if` statement


It is ok:
```javascript
    return value !== undefined ? result + value : result;
```
It is ok, too:
```javascript
    result = value !== undefined ? result + value : result;
```

There were not suggestions to apply, only valid examples

Reply:
replaced to standart if  view )